code/modules/projectiles/gun_attachables.dm

```diff
@@ -94,10 +94,7 @@ Defined in conflicts.dm of the #defines folder.
 
 /obj/item/attachable/attackby(obj/item/I, mob/user)
 	if(flags_attach_features & ATTACH_RELOADABLE)
-		if(user.get_inactive_hand() != src)
-			to_chat(user, SPAN_WARNING("You have to hold [src] to do that!"))
-		else
-			reload_attachment(I, user)
+		reload_attachment(I, user)
```

Comment:
this will make attachments on guns reloadable when not in your offhand, which is not in the changelog. Remove.